Gruntfile.coffee

```diff
@@ -1009,6 +1009,7 @@ module.exports = (grunt) ->
 						"W010" # Using `.pull-left` or `.pull-right` as part of the media object component is deprecated as of Bootstrap v3.3.0. Use `.media-left` or `.media-right` instead.
 						"E013" # Only columns (`.col-*-*`) may be children of `.row`s
 						"E014" # Columns (`.col-*-*`) can only be children of `.row`s or `.form-group`s
+						"E017" # Support for checkboxes with explicit labels instead of implicit labels
```

Comment:
We'll have to look into a new rule for this use case. (checkbox AND radio included)